java/ql/src/Metrics/Summaries/GeneratedVsManualCoverage.ql

```diff
@@ -0,0 +1,64 @@
+/**
+ * @id java/summary/generated-vs-manual-coverage
+ * @name Metrics of generated versus manual MaD coverage
+ * @description Expose metrics for the number of API endpoints covered by generated versus manual MaD models.
+ * @kind table
+ * @tags summary
+ */
+
+import java
+import semmle.code.java.dataflow.FlowSummary
+import utils.modelgenerator.internal.CaptureModels
+
+/**
+ * Returns the number of `DataFlowTargetApi`s with Summary MaD models
+ * for a given package and provenance.
+ */
+bindingset[package]
+private float getNumMadModeledApis(string package, string provenance) {
+  provenance in ["generated", "manual", "both"] and
```

Comment:
You can get rid of the bindingset and the `provenance in ["generated", "manual", "both"]` by changing `count` to `strictcount`.

Reply:
True, but would it be useful at all to see the packages that have a count of 0 for one or more columns?  If not, I'll switch back to `strictcount`.

Reply:
> would it be useful at all to see the packages that have a count of 0 for one or more columns?

Yes.

Reply:
Okay, thanks, I'll leave it as `count` then.

Reply:
Ah, right. Makes sensse to keep as-is, then.